src/main/java/org/terasology/metalrenegades/world/dynamic/MRBiome.java

```diff
@@ -76,6 +99,14 @@ public Block getBelowSurfaceBlock(Vector3ic pos, float density) {
         }
     }
 
+    private Block getStratum(Vector3ic pos) {
+        // Generate a strata index with a modulus of the y coordinate, adjusted with noise
+        // Strata are 20 (1/0.05) blocks high on average, but it varies since the noise value depends on height
+        float noise = strataNoise.noise(pos.x(), pos.y(), pos.z());
+        int idx = (int) Math.abs((float) pos.y() * 0.05 + noise * rocks.length + 1000) % rocks.length;
```

Comment:
As I understand, the first term `pos.y() * 0.05`, handles the changing of the modulus for every 20 blocks. The term `noise * rocks.length` is there to bring in some variation. The `+ 1000` exists to make the term positive before it's plugged into the modulus. I'm confused as to why there is a `Math.abs` again on the outside. To handle negative modulus values, at least in C++, I've seen people use something along the lines of 
```
int idx = (int) ((float) pos.y() * 0.05 + noise * rocks.length) % rocks.length;
if(idx<0) idx+= rocks.length;
```
So have a look at that if you think that works better/ is more readable.

Secondly, it's not clear as to why in the term `noise * rocks.length` the noise is scaled by the number of strata layers. If I'm not wrong, this term controls the amplitude of the line between different strata, So `noise * 10` would cause that line to be a wave with a larger amplitude compared to `noise * 2`. I would expect it to be scaled by a constant instead. 

Lastly, I was wondering if it would be possible to have different heights for different strata layers. I think this would be useful because having equally large sections of both dark and light blocks leads to very high contrast to the scenery. We could try experimenting with having deep layers of light blocks, with short layers of dark blocks in between and see if that  looks better.